application/templates/_macros/_details.html

```diff
@@ -0,0 +1,13 @@
+{# Adapted from https://github.com/alphagov/govuk-frontend/blob/master/src/components/details/template.njk #}
+{% macro govukDetails(summaryHtml=None,summaryText=None,text=None,html=None,id=None,attributes={},open=False,classes=None) %}
+  <details {%- if id %} id="{{id}}"{% endif %} class="govuk-details {%- if classes %} {{ classes }}{% endif %}" {%- for attribute, value in attributes.items() %} {{attribute}}="{{value}}"{% endfor %} {{- " open" if open }}>
+  <summary class="govuk-details__summary">
+    <span class="govuk-details__summary-text">
+      {{ summaryHtml | safe if summaryHtml else summaryText }}
```

Comment:
We shouldn't have the `| safe` on any field that renders user-supplied input, as it opens us up to cross-site scripting type attacks. (Yes, I know only people we trust have access, but it's still definitely a security hole that would fail a pen test).  We currently have `render_markdown` - would that do the job here?